upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -107,20 +108,128 @@ backup of Workload Manager configuration data and files is created. Once complet
+traffic. CSM 1.2 introduces bifurcated CAN (BICAN), which is designed to separate admin network traffic and user network traffic.
+With BICAN, the pre-1.2 CAN network is split into two separate networks:
+
+1. Customer Management Network (CMN)
+
+   This network allows only system administrative access from the customer site. The pre-1.2 CAN is renamed to CMN. By
+   the end of the CSM 1.2 upgrade, all non-administrative access, such as from UANs, will be removed from CMN.
+
+   During the CSM 1.2 upgrade, UANs will retain their pre-1.2 CAN IPs in order to minimize disruption to UANs. However,
+   toward the end of the CSM 1.2 upgrade, UANs will stop registering themselves on CMN and will receive new IPs on the
+   CAN/CHN network below. This process is described in more detail in [UAN Migration](#uan-migration).
+
+1. Customer Access Network (CAN) / Customer High-speed Network (CHN)
+
+   For user traffic only (e.g. users running and monitoring jobs), CSM 1.2 allows you to pick one of two networks:
+
+      - Customer Access Network (CAN): this is a new network (VLAN6 in switches) that runs over the management network. This
+         network must not be confused with pre-1.2 CAN, which was a monolithic network that allowed both user and administrative
+         traffic, was configured as VLAN7 in switches, and is now renamed to CMN. The new CAN allows only user traffic.
+
+      - Customer High-speed Network (CHN): this is a new network (VLAN5 in switches) that runs over the high-speed fabric.
+
+   You must only pick either the new CAN or CHN, but not both. The rest of the installation guide will provide you with
+   options for configuring either the new CAN or CHN.
+
+Pivoting the pre-1.2 CAN to the new CMN allows administrative traffic (already on the pre-1.2 CAN) to remain as-is while
+moving standard user traffic to a new site-routable network.
+
+### UAN migration
+
+Certain steps are taken in order to minimize disruption to UANs during the CSM 1.2 upgrade process:
+
+1. During the upgrade, the switch `1.2 Preconfig` will not remove UAN ports from the CMN VLAN (the pre-1.2 CAN), allowing UANs
+   to retain their existing IPs during the CSM 1.2 upgrade process. Traffic to and from UANs will still flow through CMN, but
+   may also flow through CAN/CHN networks if desired.
+
+1. CFS will be temporarily disabled for UANs, so that running CFS plays does not remove CMN interfaces from UANs. As mentioned
+   in [Abstract (Stage 0.3)](#abstract-stage-03), network configuration is controlled by data in SLS, but CFS plays also pick up
+   the same SLS data, which can lead to UANs being prematurely removed from CMN and causing UAN outage. As such, CFS plays
+   need to be disabled for UANs.
+
+   To disable CFS plays for UANs, you must remove CFS assignment for UANs by running the following command:
+
+   ```bash
+   what-is-the-command-to-disable-CFS-for-UANs.sh
```

Comment:
```
export CRAY_FORMAT=json
for xname in $(cray hsm state components list --role Application --type node | jq -r .Components[].ID)
do
    cray cfs components update --enabled false --desired-config "" $xname
done
```

Reply:
To be specific, the first comment is all "Application" type nodes. To further restrict this to only Application nodes that are sub-type UAN:
```
export CRAY_FORMAT=json
for xname in $(cray hsm state components list --role Application --subrole UAN --type node | jq -r .Components[].ID)
do
    cray cfs components update --enabled false --desired-config "" $xname
done
```
Imo, using only `--role Application` is likely best. Maybe we could note that `--sub-role UAN` could be used to only disable CFS for UANs.

Reply:
Thanks for that pointer. Are CFS plays different for other types of application nodes? If they are, then maybe we should stick with UANs only to be more consistent between the warning and the oops-I-did-it-now-what. I will, however, add some words in the beginning of this section that even though this section talks about UANs, it really applies to all types of application nodes, as well as a warning below this snippet that this is for UANs only.

Reply:
Made the changes as per my last comment. Also, added `Note that these steps apply to all types of application nodes and not just UANs -- the term "UAN" just happens to be more commonly used and understood when referring to user activity.` at the start of the `UAN Migration` section.